prometheus/push/push.go

```diff
@@ -197,7 +208,7 @@ func (p *Pusher) push(method string) error {
 		return err
 	}
 	buf := &bytes.Buffer{}
-	enc := expfmt.NewEncoder(buf, expfmt.FmtProtoDelim)
+	enc := expfmt.NewEncoder(buf, p.expfmt)
```

Comment:
`p.expfmt` has to be used, too, down in line 236, instead of the hardcoded `expfmt.FmtProtoDelim`.

Reply:
good catch, updated